docs/cluster.md

```diff
@@ -0,0 +1,195 @@
+      JF_HOST: "localhost:5002"
+      JF_PORT: 5002
+      JF_METRICS_PORT: 6002
+      JF_DIST_NODE_NAME: j2@jellyfish2
+    ports:
+      - 5002:5002
+      - 6002:6002
+
+networks:
+  net1:
+    driver: bridge
+```
+
+Because we run Jellyfishes in the same Docker network:
+* we can use `JF_DIST_NODE: "sname"`, which allows
+us to reference Jellyfishes using their service names so
+`jellyfish1` and `jellyfish2`
+* we don't need to export EPMD (`4369`) or distribution (`9000`)
+ports
+
+
+### Running with Docker (globally)
+
+When forming a cluster across multiple machines:
+* you have to take care of [Extra Network Configuration](#configuring-a-cluster)
+* you also can't use `JF_DIST_MODE="sname"` as you have to name Jellyfish nodes using their publicly available IP address
+or domain names (see `JF_DIST_NODE_NAME` and `JF_DIST_MODE`)
+* you can't simulate this setup locally as you won't be able to expose two EMPD ports on the same machine.
+See [Deeper dive into Erlang Distribution](#deeper-dive-into-erlang-distribution) for more information.
+
+See our [Jellyfish Videoroom deployment configuration](https://github.com/jellyfish-dev/jellyfish_videoroom/blob/main/docker-compose-deploy.yaml) for an example.
+
+## Verifying that a cluster has been created
+
+When a cluster is created correctly you will see a log indicating that one of your Jellyfishes connected to the other one, e.g.:
+
+```
+[info] [libcluster:epmd_cluster] connected to :"j1@127.0.0.1"
+```
+
+To see clustering in action, create two rooms using the same Jellyfish address and observe how they are load balanced:
+
+```bash
+curl -X POST -H "authorization: Bearer development" http://localhost:4002/room
+# {"data":{"jellyfish_address":"localhost:4001"}}
+```
+
+```bash
+curl -X POST -H "authorization: Bearer development" http://localhost:4002/room
+# {"data":{"jellyfish_address":"localhost:4002"}}
+```
+
+Load balancing can also be tested using [Jellyfish Dashboard](https://github.com/jellyfish-dev/jellyfish-dashboard).
+See [here](./tutorials/dashboard#loadbalancing-in-dashboard) for more information.
+
+## Deeper dive into Erlang Distribution
+
+The most tricky thing when running Jellyfish in a cluster is to understand the role of EPMD.
+EPMD is responsible for mapping node names to ports they use.
+Node names consist of two parts `nodename@hostname`.
+`hostname` identifies a host machine, and `nodename` identifies a node on this machine.
+When node A wants to connect to node B, it uses node's B name to ask EPMD (running on node B) 
+for the actual port node B is accessible on.
+
+**Example**
+
+1. Let's assume that node's B name is `nodeb@192.168.1.2`.
+1. Node A will contact EPMD using `192.168.1.2` IP address and port `4369`, and ask for the
+port number used by a node with the name `nodeb`.
+1. After that, node A can contact node B using `192.168.1.2` IP address and port returned by EPMD.
+
+A couple of notes about EPMD:
+* it always uses port `4369` (TCP)
```

Comment:
I would say that by default, it uses 4369.

Reply:
Currently there is no way to configure this port